lib/src/boringssl/lookup/utils.dart

```diff
@@ -101,14 +101,7 @@ Uri? _findDotDartTool() {
   //       `.dart_tool/package_config.json` file.
```

Comment:
This comment is misleading. dart:isolate is available, but the function for getting the packageConfig is async...